virttest/utils_net.py

```diff
@@ -3026,7 +3026,12 @@ def get_host_ip_address(params):
         raise NetError("Error while retrieving IP address for host.")
     logging.warning("No IP address of host was provided, using IP address"
                     " on %s interface", str(net_dev))
-    host_ip = get_ip_address_by_interface(net_dev)
+    # host_ip = get_ip_address_by_interface(net_dev)
```

Comment:
You can safely remove line with commented out function.
Commenting out code is a bad approach for public code.
You always can get previous version using git.